reviewrot/githubstack.py

```diff
@@ -159,9 +159,13 @@ def get_reviews(self, uname, repo_name, age=None, show_last_comment=None):
                     )
                     continue
 
+            title = pr.title
+            if pr.draft:
+                title = f"Draft: {title}"
```

Comment:
Could use the `WIP:` prefix instead so the UI also classifies it correctly. Maybe something like this:
```python
title = pr.title
if pr.draft and not title.upper().startswith("WIP"):
    title = "WIP: " + title
```

Reply:
Good point, updated. "Draft:" would also work I think but it is OK to use perhaps more common "WIP".

Reply:
Just `WIP`: https://github.com/redhat-aqe/review-rot/blob/3436ce9b9fd4d6aaba3fe4f4dc707cc040b85f56/web/js/site.js

Reply:
Oh, I see, I forgot about the UI part.